server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesHotThreadsAction.java

```diff
@@ -60,6 +67,12 @@ public String getName() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
+        if (request.rawPath().equals("/_cluster/nodes/hotthreads") 
```

Comment:
This complexity isn't necessary. You can override the `deprecatedRoutes` method.

Reply:
I see. Thanks!

Reply:
I have modified this part. Thanks!